javascript/autoFillSearchParams.js

```diff
@@ -0,0 +1,45 @@
+class AutoFillSearchParams {
+    searchParams = new URLSearchParams(decodeURIComponent(location.search));
+    result = '';
+    paramsKeys = ['prompt', 'negative_prompt', 'size', 'seed', 'steps', 'sampler', 'cfg', 'clip_skip', 'batch_size', 'ensd']
+
+    joinParams(key, value) {
+        const joinValue = `${key.replace('_', ' ').replace(key[0],key[0].toUpperCase())}:${value}`;
```

Comment:
Why not put this line behind the two if conditions. Since if any of the if conditions is true, then this line is no longer nacessary.